src/style.css

```diff
@@ -1,4 +1,40 @@
-/*
-  Tady nic není, ale by být mělo.
-  Sem vlož CSS z html-vzoru, které je společné pro celou aplikaci
-*/
\ No newline at end of file
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display:wght@700&family=Poppins:wght@300;400;700&display=swap');
+
+html {
+	box-sizing: border-box;
+}
+
+*,
+::before,
+::after {
+	box-sizing: inherit;
+}
+
+body {
+	font-family: 'Poppins', sans-serif;
+	font-weight: 300;
+	font-size: 18px;
+	line-height: 1.5;
+	color: #333232;
+	background: #fff9f1;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+	font-family: 'Playfair Display', serif;
+}
+
+.container {
+	max-width: 900px;
+	margin: 0 auto;
+}
+
+.products {
+	display: grid;
+	grid-template-columns: repeat(3, 1fr);
+	gap: 30px;
+}
```

Comment:
stačí to mít tady, můžeš to na všech ostatních místech vymazat. Ještě se o tom bude Luděk bavit, ale ty styly jak je používáme teď jsou globální, tj ať už do jakéhokoli stylu dáš jakoukoli classu, tak se aplikuje kdekoli v projektu.

Ale dělit styly pro komponentu je dobrý pattern a tak ho luděk už chce ukazovat od začátku :)

kdyby ses o stylech (v reactu) chtěla dozvědet víc a dřív, tak řekni a najdeme něco a můžeme třeba zkusit na příští/popříští úkol